### PR TITLE
[doc] PR for Add docs to universe troubleshooting for follower lag check during upgrades PLAT-6883

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6883

### CODE REVIEW

**Comment by lizayugabyte** [comment]
@charleswang234 Could you review this doc PR, please?

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 1]
```diff
@@ -427,4 +427,24 @@ The value of `core_pattern` can be a literal path or it can contain a pipe symbo
...
+
+## Perform the follower lag check during upgrades
+
+You can use the follower lag check to ensure that the YB-Master or YB-TServer process is caught up to its peers. To find this metric on Prometheus, execute the following:
+
+```sh
+max by (instance) (follower_lag_ms{instance='<ip>:<http_port>'})
+```
+
+- *ip* represents the YB-Master IP (default: 7000) or the YB-TServer IP (default: 9000).
```
Comment (charleswang234) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:440]: The (default: 7000) and (default:9000) should be moved below to the HTTP port line below

Ex:  *http_port* represents the HTTP port on which the YB-Master (default: 7000) or YB-TServer (default: 9000) is listening. 

Mistake on my end for the comment on Jira

**Review by charleswang234 — COMMENTED** [review]
Overall, looks good! Left a couple comments! Thanks for adding the docs for this!

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 2]
```diff
@@ -427,4 +427,24 @@ The value of `core_pattern` can be a literal path or it can contain a pipe symbo
...
+
+Typically, the maximum follower lag of a healthy universe is a few seconds at most. The following reasons may contribute to a significant increase in the follower lag, potentially reaching several minutes:
+
+- Node issues, such as network problems between nodes, an unhealthy state of nodes, or inability of the node's YB-Master or YB-TServer process to properly serve requests. The lag usually persists until the issue is resolved.
+- Issues during a rolling upgrade, when the YB-Master or YB-TServer process is stopped, upgrade on the associated process is performed, then the process is restarted. During the downtime, writes to the database continue to occur, but the associated YB-Master or YB-TServer are left behind. The lag gradually decreases after the YB-Master or YB-TServer has restarted and can serve requests again. However, if an upgrade is performed on a universe that is not in a healthy state to begin with (for example, a node is down or is experiencing an unexpected problem), a failure is likely to occur. To remedy the situation, perform the following:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:448]: Added info

[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md]: Added info

**Pull request merged**